Userland/Libraries/LibWeb/CSS/ResolvedCSSStyleDeclaration.cpp

```diff
@@ -229,7 +229,8 @@ static NonnullRefPtr<StyleValue const> style_value_for_size(Size const& size)
         return IdentifierStyleValue::create(ValueID::MinContent);
     if (size.is_max_content())
         return IdentifierStyleValue::create(ValueID::MaxContent);
-    // FIXME: Support fit-content(<length>)
+    if (size.is_fit_content())
+        return IdentifierStyleValue::create(ValueID::FitContent);
```

Comment:
This doesn't quite resolve the FIXME. It can either be `fit-content`, or `fit-content(<length-percentage>)`. Though it looks like we only currently support the identifier version.

Reply:
Hmmm so should I put the FIXME back? or is this fine as-is

Reply:
Yeah adding the FIXME back seems good.